clients/src/main/java/org/apache/kafka/clients/FetchSessionHandler.java

```diff
@@ -47,7 +47,7 @@
  * and over, without explicitly enumerating all the partitions in the request and the
  * response.
  *
- * FetchSessionHandler tracks the partitions which are in the session.  It also
+ * FetchSessionHandler tracks the partitions which are in the session. It also
```

Comment:
Just to clarify. To put two blanks between two sentences is a writing style that does back to classic type writers -- having a larger space between sentences makes it easier to read. Hence, removing all those double-blanks (to correct writing style) is just a lot of "noise" with no real value...

We can still merge this PR (as you put a lot of work into it already), however, I would encourage you to not do such PRs in the future. It's just additional review load (and committers already cannot handle the current load...) with no real value and thus a "waste" of your own time as well as the reviewer's time.

Reply:
@mjsax I see. I’ll be more careful from now on. Thanks for the detailed explanation.